components/main-with-sidebar/main-with-sidebar.html

```diff
@@ -0,0 +1,33 @@
+<div class="mzp-l-content mzp-has-sidebar mzp-l-sidebar-left">
+  <main class="mzp-l-main">
+    <p>Main content.</p>
+    <p>The sidebar comes <strong>after</strong> the main section in the source order.</p>
+  </main>
+
+  <aside class="mzp-l-sidebar">
+    <p>Sidebar content.</p>
+  </aside>
+</div>
+
+<div class="mzp-l-content mzp-has-sidebar mzp-l-sidebar-right">
+  <main class="mzp-l-main">
+    <p>Main content.</p>
+    <p>The sidebar comes <strong>after</strong> the main section in the source order.</p>
+  </main>
+
+  <aside class="mzp-l-sidebar">
+    <p>Sidebar content.</p>
+  </aside>
+</div>
+
+<!-- Inline styles are for demo purposes only. Prefer external stylesheet in production. -->
```

Comment:
Completely arbitrary colors here. Also, not sure what to do with variants. It requires a right or left layout, so neither is really "default" and it felt important to display them both, but that kind of breaks the render handle functionality, so maybe not worth it? Or move Right Sidebar to a variant? Very open to recommendations here.

Reply:
Definitely important to display them both! However, going with how Craig setup the picto variants (http://localhost:3000/components/detail/picto--default), each type of picto/side of picto has its own variant to be shown. 
It would have probably been better to have the LTR/RTL options like we do in Drizzle, but we don't have that set up just yet. So I would say just stick to the variants to make it more consistent with how other components are laid out.

Reply:
Also, TIL the CSS color `papayawhip`. Nice 🍦

Reply:
> It would have probably been better to have the LTR/RTL options like we do in Drizzle

This wouldn't entirely do what we need here, which is show that there are two layout classes available. The LTR or RTL toggle would only show how those classes respect the language direction. (i.e. `mzp-l-sidebar-right` would move to the the left side on a RTL direction)

re: consistency with variant use 💯 - will update

Reply:
Ah, yes that's a reminder I needed.